.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build and publish Develepment Charm
+on:
+  push:
+    branches:
+      - develop
```

Comment:
I'm thinking maybe this should be possible to run on any branch? So that it is not hard coded? Does that make sense?